src/context/analysisResultStateSlice.ts

```diff
@@ -217,11 +218,23 @@ export const requestAndStoreExposedPopulation = createAsyncThunk<
     wfsParams,
   );
 
-  const features = (await fetchApiData(
+  const apiFeatures = (await fetchApiData(
     ANALYSIS_API_URL,
     apiRequest,
   )) as Feature[];
 
+  const { scale, offset } = populationLayer.wcsConfig ?? {
+    scale: undefined,
+    offset: undefined,
+  };
+
+  const features =
+    !scale && !offset
+      ? apiFeatures
+      : apiFeatures.map(f =>
+          scaleFeatureStat(f, scale || 1, offset || 0, statistic),
```

Comment:
we could technically just give `apiFeatures.map(f =>
          scaleFeatureStat(f, scale || 1, offset || 0, statistic)` here without a ternary right?
Since the `|| 1` and `|| 0` would anyway negate the effect of undefined scale or offset. The only improvement I see over this is a small performance boost. Let me know what you think :+1:

Reply:
Yes. Basically is to avoid the loop if scale and offset are not defined